### ISSUE TITLE
AWS OpsWorks resources cannot be imported from any region other than us-east-1 

### ISSUE THREAD

**Issue by blooshirt** [hashicorp/terraform#9826]
This issue arrises from a question on the mailing list (see [here](https://groups.google.com/forum/#!topic/terraform-tool/GOckyBRzX2Y)). Here, it was confirmed that this is a bug, thus this issue.

The problem is that OpsWorks resources cannot be imported from anywhere other than us-east-1. The reason for this is that the OpsWorks client is hard-coded to connect to us-east-1 [here](https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/config.go#L278). The assumption that OpsWorks is only supported in us-east-1 was true until [recently](https://aws.amazon.com/blogs/devops/aws-opsworks-endpoints-available-in-11-regions/). It is now incorrect.

One possible solution is to remove the restriction of the OpsWorks client to us-east-1.

### Terraform Version
Terraform v0.7.8

### Affected Resource(s)
I have confirmed that the issue affects:
- aws_opsworks_stack
- aws_opsworks_custom_layer

However, it's likely that this affects all OpsWorks resources.

### Terraform Configuration Files
None necessary

### Debug Output
https://gist.github.com/blooshirt/269a2152eac887b2e66f11a787211926

### Expected Behavior
Terraform should import the OpsWorks stack with the given ID.

### Actual Behavior
Terraform fails with an error indicating that the stack cannot be found.

### Steps to Reproduce
1. Create test stack on AWS in eu-west-1:
````
aws opsworks create-stack \
        --name terraform_test \
        --vpc-id <vpc_id> \
        --service-role-arn <service_role_arn> \
        --default-instance-profile-arn <ec2_role_instance_profile> \
        --stack-region eu-west-1 \
        --default-subnet-id <subnet_id> \
        --configuration-manager Name=Chef,Version=12
````
2. Import stack with Terraform:
````
TF_LOG=DEBUG terraform8 import aws_opsworks_stack.terraform_test <stack_id>
````
3. Input eu-west-1 when prompted for region.
4. Observe failure

**Comment by steveh** [comment]
I've noticed that new Stacks always get created in the `us-east-1` region; I suspect fixing this will fix that too. 

The `stack-region` parameter of http://docs.aws.amazon.com/cli/latest/reference/opsworks/create-stack.html

**Comment by chriswessells** [comment]
I edited the file mentioned in blooshirt's reporting of the bug and compiled in debug mode.  I was able to successfully run terraform import was_opsworks_stack for zone us-west-1.

Unfortunately when I tried to run the tests they failed, so I couldn't issue a pull request. 

What I did:
this is line 285
client.opsworksconn = opsworks.New(usEast1Sess)

I changed it to 
client.opsworksconn = opsworks.New(less)

So it uses the region passed in by the command line

I don't know go, so if I get ambitious I may try and fix the tests and issue a pull request.

Thanks,
